packages/quantic/force-app/main/default/lwc/quanticSmartSnippetSource/quanticSmartSnippetSource.css

```diff
@@ -0,0 +1,8 @@
+.smart-snippet__source-uri {
+  color: #1a1b1e;
+  font-weight: 400;
+}
+
+.smart-snippet__source-title {
+  line-height: 28px;
```

Comment:
would you mind filling me in on why we need this specific line-height?

Reply:
This specific line-height was used in the mockups and it's different than default line height that comes with the SLDS classes, do you think we should only stick to the default styling of the SLDS text classes?

Reply:
Maybe it's just me but I prefer using margin and padding over altering line-height. In my experience line-height and font-size have a tight relationship and given font-size is something we have less control over unless specified, I'm cautious about it. Maybe overly cautious.

Reply:
Makes sense👌 I'll update this and use the SLDS padding classes instead.